development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -56,7 +56,7 @@ SCons is invoked by just calling ``scons``.
 However, this will do nothing except list the available platforms, for
 example:
 
-::
+.. code-block:: none
```

Comment:
Same here, shell highlighting is bogus so best just remove it.

Reply:
That's not any shell syntax, but free terminal text. The highlighting is then either ignored or wrong:
![bildschirmfoto](https://user-images.githubusercontent.com/4047289/33288527-e4fb3778-d3bc-11e7-8c54-c82e6ce65aea.png)

Reply:
That's why it's better to remove the highlighting for the whole page than override it everywhere. Godot's default highlighting is gdscript, so you should use `.. highlight:: none`.

Reply:
Sure, I didn't know that.